helpme/models.py

```diff
@@ -61,7 +61,7 @@ def been_updated(self):
 ##################
 
 class Category(models.Model):
-    category = models.CharField(max_length=120)
+    category = models.CharField(_("Category"), max_length=120)
```

Comment:
Do any of the other fields need to be localized?

Reply:
@renderbox I don't think they're included anywhere on the site at the moment, since we removed them from the form

Reply:
If you have time it's good to add them in because the Django Admin will make use of them too.